2023-05-26 15:48:26,380 INFO    MainThread:13832 [wandb_setup.py:_flush():76] Current SDK version is 0.15.3
2023-05-26 15:48:26,380 INFO    MainThread:13832 [wandb_setup.py:_flush():76] Configure stats pid to 13832
2023-05-26 15:48:26,380 INFO    MainThread:13832 [wandb_setup.py:_flush():76] Loading settings from C:\Users\ASUS\.config\wandb\settings
2023-05-26 15:48:26,381 INFO    MainThread:13832 [wandb_setup.py:_flush():76] Loading settings from C:\Users\ASUS\Untitled Folder\Sample-TA\wandb\settings
2023-05-26 15:48:26,381 INFO    MainThread:13832 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-05-26 15:48:26,381 INFO    MainThread:13832 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-05-26 15:48:26,381 INFO    MainThread:13832 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-05-26 15:48:26,382 INFO    MainThread:13832 [wandb_init.py:_log_setup():507] Logging user logs to C:\Users\ASUS\Untitled Folder\Sample-TA\wandb\run-20230526_154826-hu4p3w9z\logs\debug.log
2023-05-26 15:48:26,382 INFO    MainThread:13832 [wandb_init.py:_log_setup():508] Logging internal logs to C:\Users\ASUS\Untitled Folder\Sample-TA\wandb\run-20230526_154826-hu4p3w9z\logs\debug-internal.log
2023-05-26 15:48:26,383 INFO    MainThread:13832 [wandb_init.py:init():547] calling init triggers
2023-05-26 15:48:26,383 INFO    MainThread:13832 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-05-26 15:48:26,383 INFO    MainThread:13832 [wandb_init.py:init():571] re-initializing run, found existing run on stack: 89lqkb93
2023-05-26 15:48:26,390 INFO    MainThread:13832 [wandb_run.py:_finish():1893] finishing run giggleeess/Sample-TA/89lqkb93
2023-05-26 15:48:26,391 INFO    MainThread:13832 [jupyter.py:save_history():445] not saving jupyter history
2023-05-26 15:48:26,391 INFO    MainThread:13832 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-05-26 15:48:26,391 INFO    MainThread:13832 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-05-26 15:48:26,392 INFO    MainThread:13832 [wandb_run.py:_atexit_cleanup():2127] got exitcode: 0
2023-05-26 15:48:26,393 INFO    MainThread:13832 [wandb_run.py:_restore():2110] restore
2023-05-26 15:48:26,394 INFO    MainThread:13832 [wandb_run.py:_restore():2116] restore done
2023-05-26 15:48:32,307 INFO    MainThread:13832 [wandb_run.py:_footer_history_summary_info():3469] rendering history
2023-05-26 15:48:32,308 INFO    MainThread:13832 [wandb_run.py:_footer_history_summary_info():3501] rendering summary
2023-05-26 15:48:32,319 INFO    MainThread:13832 [wandb_run.py:_footer_sync_info():3428] logging synced files
2023-05-26 15:48:32,530 INFO    MainThread:13832 [wandb_init.py:init():596] starting backend
2023-05-26 15:48:32,530 INFO    MainThread:13832 [wandb_init.py:init():600] setting up manager
2023-05-26 15:48:32,535 INFO    MainThread:13832 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-05-26 15:48:32,542 INFO    MainThread:13832 [wandb_init.py:init():606] backend started and connected
2023-05-26 15:48:32,608 INFO    MainThread:13832 [wandb_run.py:_label_probe_notebook():1238] probe notebook
2023-05-26 15:48:32,609 INFO    MainThread:13832 [wandb_run.py:_label_probe_notebook():1248] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-05-26 15:48:32,609 INFO    MainThread:13832 [wandb_init.py:init():700] updated telemetry
2023-05-26 15:48:32,730 INFO    MainThread:13832 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2023-05-26 15:48:34,044 INFO    MainThread:13832 [wandb_run.py:_on_init():2177] communicating current version
2023-05-26 15:48:34,657 INFO    MainThread:13832 [wandb_run.py:_on_init():2186] got version response 
2023-05-26 15:48:34,657 INFO    MainThread:13832 [wandb_init.py:init():787] starting run threads in backend
2023-05-26 15:49:02,693 INFO    MainThread:13832 [wandb_run.py:_console_start():2158] atexit reg
2023-05-26 15:49:02,694 INFO    MainThread:13832 [wandb_run.py:_redirect():2013] redirect: SettingsConsole.WRAP_RAW
2023-05-26 15:49:02,694 INFO    MainThread:13832 [wandb_run.py:_redirect():2078] Wrapping output streams.
2023-05-26 15:49:02,694 INFO    MainThread:13832 [wandb_run.py:_redirect():2103] Redirects installed.
2023-05-26 15:49:02,696 INFO    MainThread:13832 [wandb_init.py:init():829] run started, returning control to user process
2023-05-26 15:49:02,697 INFO    MainThread:13832 [wandb_config.py:__setitem__():151] config set max_len = 13 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001B2232C5DC0>>
2023-05-26 15:49:02,698 INFO    MainThread:13832 [wandb_run.py:_config_callback():1286] config_cb max_len 13 None
2023-05-26 15:49:02,700 INFO    MainThread:13832 [wandb_config.py:__setitem__():151] config set buckets = 50 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001B2232C5DC0>>
2023-05-26 15:49:02,700 INFO    MainThread:13832 [wandb_run.py:_config_callback():1286] config_cb buckets 50 None
2023-05-26 16:20:24,754 WARNING MsgRouterThr:13832 [router.py:message_loop():77] message_loop has been closed
